### PR TITLE
[BestPractices] restructured text format for the installation instructions template

### PR DESCRIPTION
**Pull request by xabbuh**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | no |
| Applies to | all |
| Fixed tickets |  |

### CODE REVIEW

File: cookbook/bundles/best_practices.rst [hunk 1]
```diff
@@ -209,52 +209,102 @@ Installation Instructions
...
+            }
 
             // ...
         }
+        ```
 
-        // ...
-    }
-    ```
+    .. code-block:: markdown
```
Comment (WouterJ) [inline cookbook/bundles/best_practices.rst]: this one has to be restructuredText?

Comment (xabbuh) [inline cookbook/bundles/best_practices.rst]: indeed

**Comment by xabbuh** [comment]
@WouterJ Though I am not sure if this can merged as is due to the changes to the `conf.py` file.

@javiereguiluz What do you think?

**Comment by javiereguiluz** [comment]
I think this is great! Thanks @xabbuh

**Comment by xabbuh** [comment]
@javiereguiluz I mean, will this work with the current config of the build server? Or do we need to wait for an update of the config before being able to merge this?

**Comment by javiereguiluz** [comment]
I've just created a pull request in the private Sphinx parser to replicate the changes you made on this PR. I think Fabien will merge them soon and then, the server will be ready to process these changes.

**Comment by xabbuh** [comment]
Thank you @javiereguiluz.

**Comment by WouterJ** [comment]
:+1: (and IIRC, the PR on symfony.com was merged?)

**Comment by javiereguiluz** [comment]
@WouterJ yes, it was merged on Saturday morning, so things should be ready by now.

**Comment by WouterJ** [comment]
Let's try it :) Thanks for creating this PR Christian!

**Comment by xabbuh** [comment]
@javiereguiluz It looks like the documentation has not been build since this merge or we don't see the build errors anymore [here](http://symfony.com/doc/build_errors).

**Pull request merged**